cmd/juju/cloud/addcredential_test.go

```diff
@@ -867,12 +871,12 @@ credentials:
 		return []params.UpdateCredentialResult{{CredentialTag: expectedTag}}, nil
 	}
 
-	stdin := strings.NewReader(fmt.Sprintf("blah\n%s\n", sourceFile))
+	stdin := strings.NewReader(fmt.Sprintf("%vblah\n%s\n", expectedStdin, sourceFile))
 
-	ctx, err := s.run(c, stdin, cloudName)
-	c.Assert(err, jc.ErrorIsNil)
+	ctx, err := s.run(c, stdin, append(args, cloudName)...)
 	c.Assert(cmdtesting.Stdout(ctx), gc.Equals, expectedStdout)
 	c.Assert(cmdtesting.Stderr(ctx), gc.Equals, expectedStderr)
+	c.Assert(err, jc.ErrorIsNil)
```

Comment:
assert err == nil needs to be before using any other return value which is usually nil if err != nil
not sure why this line was moved?

Reply:
OH I was using this line in my tests.... Forgot to move back :) will now.